src/cmds/net/telnetd/telnetd.c

```diff
@@ -331,6 +332,12 @@ static void *telnetd_client_handler(void* args) {
 				MD(printf("read on sock: %d %d\n", sock_data_len, errno));
 			}
 			sock_data_len = telnet_fix_crnul(s, sock_data_len);
+
+			/* TODO: Check if there's T_IAC in string and delete it and
+					 following commands, instead of dropping whole message */
+			if (strchr((const char *)s, T_IAC) != NULL)
```

Comment:
Probably, it would be useful to add at least `log_debug` here, to notify about options skipping.